### ISSUE TITLE
got a bus error on raspberry pi

### ISSUE THREAD

**Issue by qibin0506** [tensorflow/tensorflow#21926]
I trained my network on my mac, and predict something on my raspberry pi with keras, but I got a bus error.

OS: Raspbian GNU/Linux 9
tensorflow: 1.9.0
keras: 2.2.2

log info:

Using TensorFlow backend.
/usr/lib/python3.5/importlib/_bootstrap.py:222: RuntimeWarning: compiletime version 3.4 of module 'tensorflow.python.framework.fast_tensor_util' does not match runtime version 3.5
  return f(*args, **kwds)
/usr/lib/python3.5/importlib/_bootstrap.py:222: RuntimeWarning: builtins.type size changed, may indicate binary incompatibility. Expected 432, got 412
  return f(*args, **kwds)
2018-08-28 14:31:25.554008: W tensorflow/core/framework/allocator.cc:108] Allocation of 63539200 exceeds 10% of system memory.
2018-08-28 14:31:26.177700: W tensorflow/core/framework/allocator.cc:108] Allocation of 63539200 exceeds 10% of system memory.
2018-08-28 14:31:26.345901: W tensorflow/core/framework/allocator.cc:108] Allocation of 63539200 exceeds 10% of system memory.
2018-08-28 14:31:36.619646: W tensorflow/core/framework/allocator.cc:108] Allocation of 63539200 exceeds 10% of system memory.
2018-08-28 14:31:36.850119: W tensorflow/core/framework/allocator.cc:108] Allocation of 63539200 exceeds 10% of system memory.
Bus error

**Comment by Anto95** [comment]
Hello, I am having the same problem. Has anyone found a solution?
I am using another network on raspberry and it works fine, but not this one.
In my case the error looks like this:


2018-08-14 09:56:59.250458: W tensorflow/core/framework/allocator.cc:108] Allocation of 5677056 exceeds 10% of system memory.
2018-08-14 09:56:59.288605: W tensorflow/core/framework/allocator.cc:108] Allocation of 5677056 exceeds 10% of system memory.
Bus error

**Comment by qibin0506** [comment]
@Anto95 Are you using keras? Maybe it's an error about keras.

**Comment by Anto95** [comment]
I'm sorry I didn't  specify.
I'm using tensorflow. The model which works is a 2D Cnn in pure tensorflow 1.9.0, the one which gives  the error is a 3D Cnn implemented with tensorflow but using Slim. Maybe that could be the problem.

**Comment by cedricve** [comment]
have the same issue @Anto95

**Comment by karmel** [comment]
@aselle - who can address Raspberry Pi issues?

**Comment by Lennart-J** [comment]
I also had this issue, using keras and tensorflow. The problem for me was that I trained my model with tensorflow 1.10 but had 1.9 on my raspberry. I only had the "Bus error" log, though
You can get a prebuilt 1.10 version for raspbian stretch from https://github.com/lhelontra/tensorflow-on-arm/releases

**Comment by tensorflowbutler** [comment]
Nagging Assignee @karmel: It has been 14 days with no activity and this issue has an assignee. Please update the label and/or status accordingly.

**Comment by tensorflowbutler** [comment]
Nagging Assignee @aselle: It has been 14 days with no activity and this issue has an assignee. Please update the label and/or status accordingly.
